tiingo/api.py

```diff
@@ -331,3 +331,66 @@ def get_bulk_news(self, file_id=None, fmt='json'):
             return data
         elif fmt == 'object':
             return dict_to_object(data, "BulkNews")
+
+    # Crypto
+    # tiingo/crypto
+    def get_crypto_top_of_book(self, tickers=[],
+                               endDate=None, exchanges=[],
+                               includeRawExchangeData=False, convertCurrency=None):
+        url = 'https://api.tiingo.com/tiingo/crypto/top'
+        params = {
+            'tickers': ','.join(tickers)
+        }
+
+        if endDate:
```

Comment:
Per @tiingo 's comment [here](https://github.com/hydrosquall/tiingo-python/pull/340/files#r334730219), it looks like the `endDate` parameter is deprecated, so I think we'd like to remove this from the arguments and the test cases.

https://api.tiingo.com/documentation/crypto

Screenshot: ![img](https://p-qKFgO2.t2.n0.cdn.getcloudapp.com/items/6quD2vb8/Image+2019-10-15+at+10.40.52+PM.png?v=062afa56ea41444d40be7e86cbd59c22)